postgis/lwgeom_sfcgal.c

```diff
@@ -45,7 +45,7 @@ Datum sfcgal_approximate_medial_axis(PG_FUNCTION_ARGS);
 Datum sfcgal_is_planar(PG_FUNCTION_ARGS);
 Datum sfcgal_orientation(PG_FUNCTION_ARGS);
 Datum sfcgal_force_lhr(PG_FUNCTION_ARGS);
-Datum sfcgal_triangulate(PG_FUNCTION_ARGS);
+Datum ST_ConstraiedDelaunayTriangles(PG_FUNCTION_ARGS);
```

Comment:
@Komzpa isn't ST_Constrai**n**edDelaunayTriangles? Missing 'n'
Thanks

Reply:
Thanks for the note!